ontology/yaml/resources/HVAC/entity_types/TRU.yaml

```diff
@@ -0,0 +1,30 @@
+# Trench Heating File for UK-LON-X1 project
+
+########################
+### Canonical Types  ###
+########################
+
+# This file defines Trench heating unit types.
+
+TRH_DFSS_DFVSC_DSP_CHWZC:
```

Comment:
This name doesn't match the types implemented below. These should match by convention.